webapp/packages/core-ui/src/useAuthenticationAction.tsx

```diff
@@ -13,15 +13,16 @@ import { useObservableRef } from '@cloudbeaver/core-blocks';
 import { useService } from '@cloudbeaver/core-di';
 
 interface IAuthenticationAction {
-  providerId: string;
+  providerId: string | undefined;
   authorized: boolean;
-  auth: () => Promise<void>;
+  isAuthorized(providerId?: string): boolean;
+  auth(providerId?: string): Promise<void>;
 }
 
 export type Options = {
-  onAuthenticate?: () => Promise<any> | void;
+  onAuthenticate?: (id: string) => Promise<any> | void;
   onClose?: () => Promise<void> | void;
-  providerId: string;
+  providerId?: string;
```

Comment:
maybe it's better to remove `providerId` & `authorized` from API and pass providerId to functions